docs/core/tutorials/top-level-templates.md

```diff
@@ -42,20 +42,38 @@ The features that make the new program simpler are *top-level statements*, *glob
 
 [Top-level statements](../../csharp/fundamentals/program-structure/top-level-statements.md) means the compiler generates the namespace, class, and method elements for your main program. You can look at the code for the new application and imagine that it contains the statements inside the `Main` method generated by earlier templates. You can add more statements to the program, just like you can add more statements to your `Main` method in the traditional style. You can even add functions. They're created as local functions nested inside the generated `Main` method.
 
-*Global `using` directives* means the compiler automatically imports a set of [`using` directives](../../csharp/language-reference/keywords/using-directive.md) based on the project type. For console applications, the following directives are implicitly included in every source file in the application:
+Both top-level statements and the [implicit `using` directives](#implicit-using-directives) simplify the code that makes up your application. To follow an existing tutorial, add any new statements to the *Program.cs* file generated by the template. You can imagine that the statements you write are between the open and closing braces in the `Main` method in the instructions of the tutorial.
+
+If you'd prefer to use the older format, you can copy the code from the second example in this article, and continue the tutorial as before.
+
+You can learn more about top-level statements in the tutorial exploration on [top level statements](../../csharp/whats-new/tutorials/top-level-statements.md).
+
+## Implicit `using` directives
+
+*Implicit `using` directives* mean the compiler automatically imports a set of [`using` directives](../../csharp/language-reference/keywords/using-directive.md) based on the project type.  For console applications, the following directives are implicitly referenced by the application:
 
 - `using System;`
+- `using System.IO;`
 - `using System.Collections.Generic;`
 - `using System.Linq;`
 - `using System.Net.Http;`
-- `using System.Net.Http.Json;`
+- `using System.Threading;`
+- `using System.Threading.Tasks;`
 
 Other application types include more namespaces that are common for those application types.
 
-These two features simplify the code that makes up your application. To follow an existing tutorial, add any new statements to the *Program.cs* file generated by the template. You can imagine that the statements you write are between the open and closing braces in the `Main` method in the instructions of the tutorial.
+### Disable implicit `using` statements
 
-If you'd prefer to use the older format, you can copy the code from the second example in this article, and continue the tutorial as before.
+If you want to turn off this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#disableimplicitnamespaceimports) in the project file.
```

Comment:
```suggestion
If you want to turn off this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#implicitusings) in the project file.
```

Reply:
Should we say instead, to turn off implicit usings, remove
```
<ImplicitUsings>enable</ImplicitUsings>
```